loaders/src/glTF/2.0/Extensions/EXT_mesh_gpu_instancing.ts

```diff
@@ -54,11 +55,6 @@ export class EXT_mesh_gpu_instancing implements IGLTFLoaderExtension {
                 return promise;
             }
 
-            // Hide the source meshes.
-            for (const babylonMesh of node._primitiveBabylonMeshes) {
-                babylonMesh.isVisible = false;
-            }
-
```

Comment:
This is still necessary right?

Reply:
No, we should not do it for thin instances.

By default, the mesh itself is no put the in the matrix buffer, so it won't be rendered. On the contrary, the mesh must have `visible` = `true` so that the frustum check returns the right answer.

Reply:
Okay, I added this because createDefaultCamera was taking the source mesh into account when it shouldn't. Please make sure the assets in https://github.com/BabylonJS/Babylon.js/issues/8149 still work properly in the sandbox.

Reply:
Yes, those two assets do display correctly.

Reply:
Hum, there are a little translated on the right compared to the online sandbox...

Reply:
Ok, the shifting occurs only when not setting `scene.useRightHandedSystem = true`. Looking for why...

Reply:
Fixed. The world matrix of the mesh has to be taken into account to calculate the global bounding info.